metabase/metabase
# Cloud migration v3

Closes https://github.com/metabase/metabase/issues/43116

### Description
Update Copy and Styling for Cloud Migration Flow

### How to verify
1. Go through Cloud Migration flow and ensure that copy / styling matches the Figma document


### Checklist
